### PR TITLE
{vis}[intel/2016a] PyFFmpeg 2.1beta + FFmpeg 0.10.16 (REVIEW)

### PR DESCRIPTION
**Pull request by boegel**

### CODE REVIEW

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
Linux SL 6.7, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/41bba6c1e22857de3a35 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
Linux centos linux 7.2.1511, Intel(R) Xeon(R) CPU E5-2660 v3 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/08e9b2e0b170454c3c7a for a full test report.

File: easybuild/easyconfigs/p/PyFFmpeg/PyFFmpeg-2.1beta-intel-2016a-Python-2.7.11.eb [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+version = '2.1beta'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'https://github.com/mhaller/pyffmpeg'
+description = """Python FFmpeg wrapper"""
+
+toolchain = {'name': 'intel', 'version': '2016a'}
+toolchainopts = {'usempi': True}
+
+source_urls = ['https://storage.googleapis.com/google-code-archive-downloads/v2/code.google.com/pyffmpeg/']
```
Comment (wpoely86) [inline easybuild/easyconfigs/p/PyFFmpeg/PyFFmpeg-2.1beta-intel-2016a-Python-2.7.11.eb]: nothing better?

**Comment by wpoely86** [comment]
lgtm

File: easybuild/easyconfigs/p/PyFFmpeg/PyFFmpeg-2.1beta-intel-2016a-Python-2.7.11.eb (hunk 1 above)
Comment (boegel) [inline easybuild/easyconfigs/p/PyFFmpeg/PyFFmpeg-2.1beta-intel-2016a-Python-2.7.11.eb]: I'm afraid not...

**Comment by boegel** [comment]
Thanks for the review @wpoely86!

**Comment by boegel** [comment]
Thanks for the review @wpoely86!

**Pull request merged**